### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
@vsalvino do you mind reviewing this? TIA!

**Comment by vsalvino** [comment]
Looks OK to me. But I'm not a maintainer of this project.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -17358,6 +17373,7 @@ inconlusive->inconclusive
 inconsisent->inconsistent
 inconsisently->inconsistently
 inconsisntency->inconsistency
+inconsistance->inconsistence
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:17376]: Suggested word `inconsistance->inconsistence` is not a valid word in some of the major online dictionaries, hunspell, aspell, Chrome browser, etc
https://www.lexico.com/search?filter=en_dictionary&query=inconsistence
https://www.ldoceonline.com/spellcheck/english/?q=inconsistence
https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=inconsistence

```text
inconsistance->inconsistency
inconsistancies->inconsistencies
```

**Review by vikivivi — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: [6e3b2cd](https://github.com/codespell-project/codespell/pull/2414/commits/6e3b2cdc7f6be58f2df27dfd175a734355743c44)

**Pull request merged**